app/src/main/java/com/keylesspalace/tusky/fragment/TimelineFragment.java

```diff
@@ -1171,7 +1183,7 @@ private void updateStatuses(List<Either<Placeholder, Status>> newStatuses, boole
 
     private void removeConsecutivePlaceholders() {
         for (int i = 0; i < statuses.size() - 1; i++) {
-            if (!statuses.get(i).isRight() && !statuses.get(i + 1).isRight()) {
+            if (statuses.get(i).isLeft() && statuses.get(i + 1).isLeft()) {
```

Comment:
using `isLeft` is better than `!isRight`

Reply:
:rofl: